src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
```

Comment:
Advice for the future, it is better to use the axios library for queries, it has more options than the built-in fetch, if you want to learn more about it, write to me